Mapify/Utils/Extensions.cs

```diff
@@ -304,6 +305,40 @@ public static string GetStationLocalizationKey(this Station station)
             return "station/" + station.stationID;
         }
 
+        public static DVLanguage GetLanguageAsDVLanguage(this TranslationSet languageSet)
+        {
+            switch (languageSet.language)
```

Comment:
The integer values for these should line up so we can just cast it directly to `DVLanguage` without manually mapping all values.

Reply:
changed